server/app/modules/DurableJobModule.java

```diff
@@ -17,6 +25,36 @@
  */
 public final class DurableJobModule extends AbstractModule {
 
+  @Override
+  protected void configure() {
+    // Binding the scheduler class as an eager singleton runs the constructor
+    // at server start time.
+    bind(DurableJobRunnerScheduler.class).asEagerSingleton();
+  }
+
+  /** Schedules the job runner to run on an interval using the akka scheduling system. */
+  public static final class DurableJobRunnerScheduler {
+
+    @Inject
+    public DurableJobRunnerScheduler(
+        ActorSystem actorSystem,
+        Config config,
+        ExecutionContext executionContext,
+        Provider<DurableJobRunner> durableJobRunnerProvider) {
+      int pollIntervalSeconds = config.getInt("durable_jobs.poll_interval_seconds");
+
+      actorSystem
+          .scheduler()
+          .scheduleAtFixedRate(
+              // Wait a random amount of time to decrease likelihood of synchronized
+              // polling with another server.
+              /* initialDelay= */ Duration.ofSeconds(new Random().nextInt(/* bound= */ 30)),
```

Comment:
Do we have an idea about how long we expect jobs utilizing this system to take?  I wonder if for some jobs, waiting up to 30 seconds might be overkill and cause a normally short-running job to take much longer.

Reply:
This is the delay from when the server starts to when it starts polling the database for jobs to run.